BOM_1_0_2a11

```diff
@@ -1,70 +1,69 @@
-                                                                                               SPDT Switch                                                                                      SWCH-08261                                                           
-1   HC4050              4504N                    DIL16                     IC4      TTL or CMOS to CMOS hex LEV  SHIFTER 
+1   74HC573N           74HC573N                DIL20                     IC1                                                                                                8-bit D latch BUS DRIVER                                                                                                                                              
+2   74LS165N           74LS165N                DIL16                     IC2, IC3                                                                                           8-bit parallel load SHIFT                                                                                                                                    
+1   AS6C1008-55PCN     AS6C1008-55PCN          128K SRAM                 U1                                                                                                 CMOS SRAM                                                           
+1   HC4050             4504N                   DIL16                     IC4                                                                                                TTL or CMOS to CMOS hex LEV  SHIFTER 
 
 MIDI Jack:
                                                                                                                                 
-4   MIDIJACK            490-SDS-50                CP-2350                   U$7, U$8, U$9, U$10  DIN 5 jack also known as MIDI jack 
+4   MIDIJACK           490-SDS-50              CP-2350                   U$7, U$8, U$9, U$10                                                                                DIN 5 jack also known as MIDI jack 
         
-
 Switches:
 
 1   EG1218              SWITCH-SPDTKIT           SWITCH-SPDT_KIT           S1 
-4   3FT1D               3FT1D                    3FT                       BUTTON1, BUTTON2, BUTTON3, BUTTON4                                                                Mulitmec - 3FT1D                                                                                                                                                      
+4   3FT1D               3FT1D                    3FT                       BUTTON1, BUTTON2, BUTTON3, BUTTON4                                                               Mulitmec - 3FT1D                                                                                                                                                      
+??? SPDT Switch SWCH-08261
```

Comment:
I don't see this in the PCB file. Replaced by EG1218?

Reply:
Hi Yatao,

I think you're looking at the older PCB design.

Just to be clear, there are two versions.

1_0_1a
1_0_2a12

The following BOM document refers to the 1_0_2a12 board.

Reply:
I'm looking at megacommand_1_0_2a12.brd, and I can see sdcard pinout & EXP_L & EXP_R (which are absent in the earlier version right?), and S1 = EG1218

Reply:
That's correct

Reply:
I can only find reference to "SWCH-08261" on the old board, which might be a place holder I used before deciding on the EG1218